### ISSUE TITLE
Only named exports may use 'export type'.

### ISSUE THREAD

**Issue by FezVrasta** [Khan/flow-to-ts#228]
With 0.1.9 the following code would be converted as follows:

JS:
```
// @flow
export type * from './types';
```

TS:
```
export * from './types';
```

But with 0.2.2 I get the following:
```
export type * from "./types";
```

which results in a TS error:

```
lib/index.d.ts:1:1 - error TS1383: Only named exports may use 'export type'.

1 export type * from "./types";
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by aminya** [comment]
Please update your TypeScript. This has been fixed.

**Comment by aminya** [comment]
Related to https://github.com/microsoft/TypeScript/issues/37238

**Comment by FezVrasta** [comment]
I'm completely TypeScript noob so sorry for the question, does `export * from './types'` work in TS or it's only exporting JS modules that way?

**Comment by aminya** [comment]
`export * from './types'` exports everything including the types. Unfortunately, TypeScript [doesn't support](https://github.com/microsoft/TypeScript/issues/37238) `export type *` now, and so `export *` should be used to solve this `flow-to-ts` issue.

**Comment by FezVrasta** [comment]
I think [this](https://github.com/Khan/flow-to-ts/commit/e4fdd54137876f87ff2ac5a73fdebcc1b7671224) broken the functionality. I'm not sure what the commit is trying to achieve considering TS doesn't support it?

**Comment by aminya** [comment]
TypeScript supports all the other syntaxes. It only doesn't support `export type *`, which is an inconsistency from the typescript compiler.

**Comment by k4b7** [comment]
@FezVrasta yeah, I messed up with that commit.  I should've converted `typeof` imports to `kind` imports.  I didn't know that `export type *` was even a thing that flow supported.  There also needs to be way more test cases for all of the different kinds of imports and exports.

**Comment by k4b7** [comment]
Please have a look at #229 and let me know if I missed any cases.

**Comment by k4b7** [comment]
I've published [v0.3.0](https://github.com/Khan/flow-to-ts/releases/tag/v0.3.0) with the import/export improvements.

**Comment by FezVrasta** [comment]
Works great, thanks!
